### DISCUSSION TITLE
Need to add a city not on our dropdown list

### DISCUSSION THREAD

**Discussion by district6wc** [code4recovery/12-step-meeting-list#1311] [Q&A] [answered]
Hi

I need to add a new city but can't seem to find where to do that, i looked at the function.php file and did not see the city listings there.
Does anyone know how to update that list for location. The city is right outside our district but we'd like to include it.

Thanks
Rose L

**Comment by anchovie91471** [comment, marked as answer]
Cities are "regions" in TSML. In the Wordpress admin, look under the Meetings and you'll see the regions menu item. You can add your city there and it will show in your dropdown.

**Comment by district6wc** [comment]
Thank you so much
Thank you.

Yours in Service,
Rose Lanaro
Saturday Morning Live, Chapel Hill

District 6 Website Coordinator Panel 62
631-525-3328
